crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -342,6 +342,16 @@ def prepare_nodes_for_os_upgrade
       prop.save
     end
 
+    # change pacemaker setting to not use pacemaker for stateless services
+    pacemaker_proposals = Proposal.all.where(barclamp: "pacemaker")
+    pacemaker_proposals.each do |proposal|
+      role = proposal.role
+      proposal.raw_data["attributes"]["pacemaker"]["clone_stateless_services"] = false
```

Comment:
can we reference the schema default here instead of hardcoding it? might be difficult..

Reply:
why the schema? the schema values are default values but here we want to make clean that we want to set this value to false explicitly.

I wonder how would that make it more clear that what we have right now.